redfishtool/Chassis.py

```diff
@@ -745,6 +747,65 @@ def clearLog(self,sc,op,rft,cmdTop=False, prop=None):
         rft.printVerbose(4,"{}:{}: in operation".format(rft.subcommand,sc.operation))
         return(8,None,False,None)
 
+    
+    def getSensors(self,sc,op, rft, cmdTop=False, prop=None):
```

Comment:
This method is not "registered" in  Chassis.runOperation() so there is no way for the user to invoke this command.